### ISSUE TITLE
No responding the UI  in first 30 seconds

### ISSUE THREAD

**Issue by ducan-ne** [shellscape/jsx-email#45]
- Component or Package Name: cli
- Component or Package Version: 2.1.1
- `@jsxp-email/cli` Version?: 1.1.1
- Operating System (or Browser): macos
- Node Version: 20.9.0
- Link to reproduction _(⚠️ read below)_:

### Expected Behavior
I'm not sure if this is an issue from jsx-email side or maybe only occur in my device, my email preview UI is not responding and I need to wait at least 30 seconds (HMR also faces this), no lagging or anything else that relate to, all js scripts also loaded.
I've read the code of jsx CLI but couldn't see any problem with the code 🥲

### Actual Behavior
Should be render fast as we run vite in normal situation

### Additional Information

Please checkout my video, I've cut last 1s when showing the UI to avoid project information, sorry about that

https://github.com/shellscape/jsx-email/assets/16853175/9503c35d-47e7-4d50-b31d-91956d9bd769

**Comment by ducan-ne** [comment]
Ah, it seems to lag a bit. I can't right-click, but the CPU number is not high.

**Comment by shellscape** [comment]
thanks for the issue. unfortunately there's not much we can do to help unless you put together a reproduction like the issue template asks for. a video isn't going to be helpful in triaging your specific setup.

**Comment by ducan-ne** [comment]
Will try to create a reproduction

**Comment by shellscape** [comment]
Awesome, thanks. Are you by chance using the `<Tailwind/>` component?

**Comment by ducan-ne** [comment]
@shellscape Yes i'm currently using <Tailwind/>, I think this might be the root cause, as I also encountered this issue with the original react-email.

Apologies for the delay in reproducing the issue. I will attempt it in a few days. However I think the problem can be <Tailwind>.

**Comment by ducan-ne** [comment]
@shellscape just tried upgrade to latest version and the performance has significantly improved. HMR is now reduced to approximately 1 second. Great job! 
Can we close this issue?

**Comment by shellscape** [comment]
Sure can, thanks for sticking with it!
